src/classes/GE_DefaultTemplate.cls

```diff
@@ -0,0 +1,234 @@
+                                                    'Billing City","required":false,"sectionId":"94d2bb2b-8cc4-3644-6ac6-d9180269c95c"},{"customLabel":"Organization Account State/Province",' + 
+                                                    '"dataImportFieldMappingDevNames":["Account1_State_Province_9db6cdbd7"],"dataType":"STRING","elementType":"field","id":' + 
+                                                    '"40cd5b93-3f6a-7be4-9e35-de769683460d","label":"Account 1: Billing State/Province","required":false,"sectionId":' + 
+                                                    '"94d2bb2b-8cc4-3644-6ac6-d9180269c95c"},{"customLabel":"Organization Account Zip/Postal Code","dataImportFieldMappingDevNames":' + 
+                                                    '["Account1_Zip_Postal_Code_4aea9a02f"],"dataType":"STRING","elementType":"field","id":"56f7dbf7-99dc-e9be-d5d8-21df79ba1621","label"' + 
+                                                    ':"Account 1: Billing Zip/Postal Code","required":false,"sectionId":"94d2bb2b-8cc4-3644-6ac6-d9180269c95c"},{"customLabel":' + 
+                                                    '"Organization Account Country","dataImportFieldMappingDevNames":["Account1_Country_1359e000c"],"dataType":"STRING","elementType":' + 
+                                                    '"field","id":"e8a74c72-6a9a-66da-3129-f48cca998fdc","label":"Account 1: Billing Country","required":false,"sectionId":"94d2bb2b-8cc4-3644-6ac6-d9180269c95c"}],' + 
+                                                    '"id":"94d2bb2b-8cc4-3644-6ac6-d9180269c95c","label":"Add or Edit Organization Account"},{"id":"1cb17425-5ced-ebd6-cdaf-44556f648e3f",' + 
+                                                    '"displayType":"accordion","defaultDisplayMode":"expanded","displayRule":"displayRule","label":"Add or Edit Primary Contact","elements":' + 
+                                                    '[{"id":"05724374-e3fa-b94b-fc6d-00dfbf358abb","label":"Contact 1: First Name","customLabel":"Contact First Name","required":false,"sectionId"' + 
+                                                    ':"1cb17425-5ced-ebd6-cdaf-44556f648e3f","defaultValue":null,"dataType":"STRING","dataImportFieldMappingDevNames":["Contact1_First_Name_73301c3c5"]' + 
+                                                    ',"elementType":"field"},{"id":"d89dbbb6-1a87-41e6-6e14-dd34a13a9274","label":"Contact 1: Last Name","customLabel":"Contact Last Name","required":false,' + 
+                                                    '"sectionId":"1cb17425-5ced-ebd6-cdaf-44556f648e3f","defaultValue":null,"dataType":"STRING","dataImportFieldMappingDevNames":["Contact1_Last_Name_563ada236"]' + 
+                                                    ',"elementType":"field"},{"id":"39051aa1-b149-6df7-6ced-02639bdacd82","label":"Address: Mailing Street","customLabel":"Contact Street",' + 
```

Comment:
@KietTruongSFDO @ptewson-sfdo @vmartinov-sforg We need clarity from PMs on how to handle the Contact address fields on the default template. These fields map to the NPSP Address object, and GEM users use the EDA Address object.

Reply:
Thanks @melissabarber - keep us posted, but in the interest of keeping this story moving, I think we need to deliver it as it was spelled out in the requirements while that conversation occurs.

Reply:
We have a meeting tomorrow to discuss. I suspect we're going with keeping the fields in for MVP and hopefully somewhat soon we're able to get farther into Mapping Sets that could provide a solution for GEM customers.